modules/apps/captcha/captcha-taglib/src/main/resources/META-INF/resources/captcha/simplecaptcha.jsp

```diff
@@ -9,11 +9,18 @@
 
 <%
 String url = (String)request.getAttribute("liferay-captcha:captcha:url");
+String portletId = (String)request.getAttribute("liferay-captcha:captcha:portletId");
+boolean headlessAPI = true;
```

Comment:
I think no need to declare a variable here.

Reply:
Fixed it. Thanks